migrate/live-migration/live-migration-from-timescaledb.md

```diff
@@ -0,0 +1,153 @@
+title: Migrate from TimescaleDB using live migration
+excerpt: Migrate from a TimescaleDB database using the low-downtime live migration method
+products: [cloud]
+keywords: [migration, low-downtime]
+tags: [migration, logical backup, replication]
+---
+
+import GettingHelp from "versionContent/_partials/_migrate_dual_write_backfill_getting_help.mdx";
+import SourceTargetNote from "versionContent/_partials/_migrate_source_target_note.mdx";
+import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
+import DumpDatabaseRoles from "versionContent/_partials/_migrate_dual_write_dump_database_roles.mdx";
+import DumpPreDataSourceSchema from "versionContent/_partials/_migrate_pre_data_dump_source_schema.mdx";
+import DumpPostDataSourceSchema from "versionContent/_partials/_migrate_post_data_dump_source_schema.mdx";
+import LiveMigrationStep2 from "versionContent/_partials/_migrate_live_migration_step2.mdx";
+
+# Live migration from TimescaleDB database with pgcopydb
+
+This document provides detailed instructions on how to perform a database migration
+from your production TimescaleDB that is either a [Managed Service for TimescaleDB] or
+self-hosted instance to Timescale in a way that requires minimal downtime
+(in the order of few minutes) on your production applications.
+
+The migration process will be carried out using the [live migration] strategy. You
+will use a live migration image that streamlines the process of replicating live
+transactions from your source database to Timescale.
+
+You should provision a dedicated instance to run the migration steps from.
+Ideally an AWS EC2 instance that's in the same region as the Timescale target service.
+For an ingestion load of 10,000 transactions/s, and assuming that the historical
+data copy takes 2 days, we recommend 4 CPUs with 4 to 8 GiB of RAM and 1.2 TiB of storage.
+
+<SourceTargetNote />
+
+In detail, the migration process consists of the following steps:
+1. Set up a target database instance in Timescale.
+1. Prepare the source database for live migration.
+1. Run the live migration image.
+1. Validate the data in target database and promote as new primary.
+
+<GettingHelp />
+
+<StepOne />
+
+## 2. Prepare the source database for live migration
+
+<LiveMigrationStep2 />
+
+<Highlight type="important">
+TimescaleDB 2.12 and above allows setting `REPLICA IDENTITY` on hypertables. However,
+if you're using a version below 2.12, it’s important that your hypertables have primary keys.
+If they don't, you will need to upgrade your TimescaleDB instance. This is because, without a
+primary key or `REPLICA IDENTITY`, operations such as `DELETE` and `UPDATE` will not be replicated,
+which will impact the consistency of your data post migration.
+</Highlight>
+
+## 3. Run live migration image
+
+First, set the database URIs as environment variables:
+
+```sh
+export SOURCE=""
+export TARGET=""
+```
+
+Next, download and run the live migration image:
+
+```sh
+docker run --rm -it \
```

Comment:
Do we not also need volume mapping to support resume?

Reply:
+1. Additionally I recommend to pass `--name live-migration`.